apps/app/src/app/knowledge-box/knowledge-box-profile/knowledge-box-profile.component.ts

```diff
@@ -17,12 +18,14 @@ export class KnowledgeBoxProfileComponent implements OnInit, OnDestroy {
   account: Account | undefined;
 
   kbForm = this.formBuilder.group({
-    title: ['', [Sluggable()]],
+    uid: [''],
+    slug: ['', [Sluggable()]],
```

Comment:
I realize that "sluggable" validator is kind of confusing. It doesn't validate if a slug is valid,
It validates if a Kb name can be automatically converted into a valid Kb slug.

Reply:
During my tests, I noticed the validator wasn't working very well indeed, or at least it didn't feel like it was working well. 
The thing is, our method `STFUtils.generateSlug` is transforming any text into a valid slug, and once saved the form is updated with the generated slug. So the current validator is good enough as it prevents the user to only enter invalid characters. For example, typing `#` is invalid because it would become an empty slug, but typing `#k` is valid because it will generate a slug `k` which is a valid one.